src/actions/historyActions.js

```diff
@@ -37,16 +37,31 @@ import { checkForMissedAssetsAction } from './assetsActions';
 import { saveDbAction } from './dbActions';
 import { getExistingTxNotesAction } from './txNoteActions';
 import { checkAssetTransferTransactionsAction } from './smartWalletActions';
+import { checkEnableExchangeAllowanceTransactionsAction } from './exchangeActions';
 import { ETH } from '../constants/assetsConstants';
 
 const TRANSACTIONS_HISTORY_STEP = 10;
 
+export const afterHistoryUpdatedAction = () => {
```

Comment:
This looks like some utility more than an action. Just as a suggestion, I wouldn't export it and probably wouldn't even call it "action", but not a big deal.

Reply:
We want to use latest and updated getState in this action after dispatch, removed the export part.